src/main/scala/no/ndla/learningpathapi/model/api/LearningPathSummary.scala

```diff
@@ -27,4 +27,5 @@ case class LearningPathSummary(@(ApiModelProperty@field)(description = "The uniq
                                @(ApiModelProperty@field)(description = "The date when this learningpath was last updated.") lastUpdated: Date,
                                @(ApiModelProperty@field)(description = "Searchable tags for the learningpath") tags: Seq[LearningPathTags],
                                @(ApiModelProperty@field)(description = "The author of this learningpath") owner: Author,
+                               @(ApiModelProperty@field)(description = "The contributors of this learningpath") copyright: Copyright,
```

Comment:
Ja, tenker det kan være greit i stedet for å lage et helt nytt opplegg for å kun ta med contributors. Tror heller ikke at det er utenkelig at det kommer spørsmål om å vise lisens i søkeresultatet heller.